haproxy/dockerdir/usr/local/bin/check_pxc.sh

```diff
@@ -12,7 +12,7 @@ MONITOR_USER='monitor'
 MONITOR_PASSWORD=$(/bin/cat /etc/mysql/mysql-users-secret/monitor)
 CUSTOM_TIMEOUT=$(/bin/cat /etc/mysql/haproxy-env-secret/HA_CONNECTION_TIMEOUT)
 OK_IF_DONOR=$(/bin/cat /etc/mysql/haproxy-env-secret/OK_IF_DONOR)
-TIMEOUT=${CUSTOM_TIMEOUT:-10}
+TIMEOUT=${CUSTOM_TIMEOUT:-$LIVENESS_CHECK_TIMEOUT}
```

Comment:
why not?
```suggestion
TIMEOUT=${LIVENESS_CHECK_TIMEOUT:-10}
```

Reply:
It is good question :) We have possibility of setting TIMEOUT via secret using CUSTOM_TIMEOUT var. That is why I left it.

Reply:
then why not
```suggestion
TIMEOUT=${CUSTOM_TIMEOUT:-${LIVENESS_CHECK_TIMEOUT:-10}}
```